core/src/main/resources/zeneventserver.conf

```diff
@@ -248,3 +248,13 @@ zep.redis.port=6379
 #zep.lucene.archive.status=READER
 #zep.lucene.archive.async.updates=false
 #zep.lucene.archive.honor.deletes=true
+
+# Zenoss Cloud configuration, disabled by default
+zep.zing.enabled=true
+zep.zing.use_emulator=true
+zep.zing.tenant=paco
```

Comment:
How will the tenant and source be coordinated with the overall CZ?

Reply:
This is one of the items in my TODO list. I will address it in a future PR.